src/config/players.json

```diff
@@ -70,6 +73,7 @@
 			],
 			"darwin": [
 				"/Applications/mpv.app/Contents/MacOS",
+				"~/Applications/mpv.app/Contents/MacOS",
```

Comment:
same here

Reply:
I changed tilde to $HOME. it's always confusing.